packages/@aws-cdk/aws-backup/lib/selection.ts

```diff
@@ -0,0 +1,154 @@
+   */
+  readonly backupSelectionSelectionId: string;
+}
+
+/**
+ * Represents the backup selection properties.
+ */
+export interface BackupSelectionProps {
+  /**
+   * The name of the selection resource.
+   */
+  readonly backupSelectionName: string;
+  /**
+   * The backup plan to assign this plan to.
+   */
+  readonly backupPlan: IBackupPlan;
+  /**
+   * Optional IAM Role to use for selecting and interacting with the AWS resources to backup.
+   * @default IAM Role based from the AWS Backup service.
+   * @see https://docs.aws.amazon.com/aws-backup/latest/devguide/iam-service-roles.html
+   */
+  readonly role?: iam.IRole;
+  /**
+   * A list of ARNs to backup.
+   * @default to no resources.
+   */
+  readonly resources?: string[];
+}
+
+/**
+ * Backup resource selection.
+ */
+export class BackupSelection extends BackupSelectionBase {
+  /**
+   * Imports a backup selection based from the given attributes.
+   */
+  public static fromBackupSelectionAttributes(scope: Construct, id: string, attrs: BackupSelectionAttributes): IBackupSelection {
+    class Import extends BackupSelectionBase {
+      public readonly backupSelectionName: string = attrs.backupSelectionName;
+      public readonly backupPlan: IBackupPlan = attrs.backupPlan;
+      public readonly backupSelectionBackupPlanId: string = attrs.backupPlan.backupPlanId;
+      public readonly backupSelectionSelectionId: string = attrs.backupSelectionSelectionId;
+      public readonly resources?: string[]; // dropped
+      public readonly role?: iam.IRole; // dropped
+    }
+
+    return new Import(scope, id);
+  }
+
+  public readonly backupSelectionName: string;
+  public readonly backupPlan: IBackupPlan;
+  public readonly backupSelectionBackupPlanId: string;
+  public readonly backupSelectionSelectionId: string;
+  public readonly role?: iam.IRole;
+  public readonly resources?: string[];
+
+  constructor(scope: Construct, id: string, props: BackupSelectionProps) {
+    super(scope, id);
+    this.backupPlan = props.backupPlan;
+    this.backupSelectionBackupPlanId = this.backupPlan.backupPlanId;
+    this.backupSelectionName = props.backupSelectionName;
+
+    const resource = new CfnBackupSelection(this, 'Resource', {
+      backupPlanId: this.backupPlan.backupPlanName,
+      backupSelection: {
+        selectionName: this.backupSelectionName,
+        iamRoleArn: this.parseRole(props).roleArn,
+        resources: props.resources,
+      }
+    });
+
+    this.backupSelectionSelectionId = resource.attrSelectionId;
+    this.backupSelectionBackupPlanId = resource.attrBackupPlanId;
+  }
+
+  private parseRole(props: BackupSelectionProps): iam.IRole {
+    if (props.role) {
+      return props.role;
+    }
+
+    const role = new iam.Role(this, 'Role', {
+      assumedBy: new iam.ServicePrincipal('backup.amazonaws.com'),
+      managedPolicies: [
+        iam.ManagedPolicy.fromAwsManagedPolicyName('service-role/AWSBackupServiceRolePolicyForBackup')
```

Comment:
Do we need the `AWSBackupServiceRolePolicyForRestores` policy as well?

Reply:
AFAIK that policy is only if a user wants to execute the restores through backup service. Since this implementation is only executing backups I'm a bit skeptical to add the policy, this would prevent unforeseen restores.
Besides that, I don't see any reason why we shouldn't add it. Let me know if you want me to add it

Reply:
what if we add a method `allowRestore` or similar and we add the policy. this would expose 1. the API and 2. explicit consent from the user to allow this.